Compute/owlbot.py

```diff
@@ -14,31 +14,30 @@
 
 """This script is used to synthesize generated parts of this library."""
 
-import synthtool as s
-import synthtool.gcp as gcp
 import logging
+from pathlib import Path
+import synthtool as s
+import subprocess
+from synthtool.languages import php
+from synthtool import _tracked_paths
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICBazel()
-common = gcp.CommonTemplates()
+src = Path(f"../{php.STAGING_DIR}/Compute").resolve()
+dest = Path().resolve()
 
-library = gapic.php_library(
-    service='compute',
-    version='v1',
-    bazel_target='//google/cloud/compute/v1:google-cloud-compute-v1-php',
-    diregapic=True
-)
+# Added so that we can pass copy_excludes in the owlbot_main() call
+_tracked_paths.add(src)
 
-# copy all src including partial veneer classes
-s.move(library / 'src')
-
-# copy proto files to src also
-s.move(
-    sources=library / 'proto/src/Google/Cloud/Compute',
-    destination='src/',
-    excludes='V*/*_*.php'
```

Comment:
I think the original code is copying all the files including partial veneer classes, so we don't need to exclude those.

Also I think we want to exclude some files in the `proto` directory. I think it's backward compatible files with underscore (e.g. `VpnGatewayStatusHighAvailabilityRequirementState_State.php`).

So maybe following?
```python
php.owlbot_main(
    src=src,
    dest=dest,
    copy_excludes=[
        src / "*/proto/src/Google/Cloud/Compute/*/*_*.php"
    ]
)
```

Reply:
I used `src / "*/proto/**/*_*.php"`

Reply:
Yeah that pattern looks great.